deepchecks/vision/dataset.py

```diff
@@ -98,17 +98,6 @@ def __init__(self,
         self._data = data_loader
         self.label_transformer = label_transformer
         self.image_transformer = image_transformer or ImageFormatter(lambda x: x)
-
```

Comment:
why delete this?

Reply:
code duplicates. Probably someone merged incorrectly